package.json

```diff
@@ -16,28 +16,28 @@
   "dependencies": {
     "is_js": "^0.9.0"
   },
-  "main": "./lib/finput.js",
+  "main": "./dist/finput.js",
   "devDependencies": {
     "babel-plugin-add-module-exports": "^0.1.2",
     "babel-preset-es2015": "^6.5.0",
     "babel-preset-stage-2": "^6.5.0",
     "babelify": "^7.2.0",
+    "browserify": "^14.4.0",
     "browserstacktunnel-wrapper": "^1.4.2",
-    "grunt": "^0.4.5",
-    "grunt-babel": "^6.0.0",
-    "grunt-browser-sync": "^2.2.0",
-    "grunt-browserify": "^4.0.1",
-    "grunt-browserstacktunnel-wrapper": "0.0.5",
-    "grunt-contrib-clean": "^1.0.0",
-    "grunt-contrib-uglify": "^0.11.0",
-    "grunt-contrib-watch": "^0.6.1",
-    "grunt-remove-logging": "^0.2.0",
-    "grunt-webdriver": "^1.0.0",
+    "budo": "^10.0.3",
+    "jasmine": "^2.4.1",
+    "mkdirp": "^0.5.1",
+    "rimraf": "^2.6.1",
+    "uglify-js": "^3.0.24",
     "jest": "^20.0.4",
     "selenium-webdriver": "^3.4.0",
     "webdriver-manager": "^12.0.6"
   },
   "scripts": {
+    "clean": "rimraf ./dist && mkdirp ./dist/",
```

Comment:
I think I might need both. `browserify` doesn't create paths that don't exist (which surprised me). So the case where `npm run build` is running for the very first time requires that something creates the path.